contributing/FORAMTTING-CODE.md

```diff
@@ -0,0 +1,169 @@
+# Formatting code blocks
+
+We want to adopt a consistent style for PowerShell code blocks and output in our documentation.
+The existing documentation has uses multiple styles and the formatting rules have changed multiple times.
+The current Open Publishing System (OPS) we use has a syntax highlighting feature that support many languages.
+OPS also adds a **Copy** button that copies the contents of the code block to the clipboard.
+This allows you to quickly paste the code into a script for testing the code example.
+However, not all examples in our documentation are intended to be run.
+Some code blocks are simple illustrations of a PowerShell concept.
+
+We are defining two scenarios for code blocks:
```

Comment:
Love the delineation here, hits the nail on the head.

Reply:
👍